src/Skia/Avalonia.Skia/FramebufferRenderTarget.cs

```diff
@@ -55,7 +55,7 @@ public IDrawingContextImpl CreateDrawingContext()
             {
                 Surface = _framebufferSurface,
                 Dpi = framebuffer.Dpi,
-                DisableTextLcdRendering = true
+                DisableSubpixelTextRendering = true
```

Comment:
IMO this should be disabled for framebuffer too (since it's used with X11 when HW-acceleration is not available) and re-enabled as a render-target property in RTB implementation here: https://github.com/AvaloniaUI/Avalonia/blob/2c3002d9bbfe887a697085b429122409ea0197a2/src/Skia/Avalonia.Skia/RenderTargetBitmapImpl.cs#L19